daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-message-ttl.md

```diff
@@ -0,0 +1,37 @@
+---
+type: docs
+title: "Pub/Sub Message's Time-To-Live"
+linkTitle: "Pub/Sub Message's TTL"
+weight: 5000
+description: "Use time-to-live in Pub/Sub messages." 
+---
+
+## Introduction
+
+In Kafka, time-to-live is configured in the topic via `retention.ms` as per [documentation](https://kafka.apache.org/documentation/#topicconfigs_retention.ms). This setup is transparent to Dapr.
```

Comment:
It feels this doc jumps in to a specific case before providing the reader more context on what this page is and what the general behavior is. If this (Kafka) is a special case I would put it further down. Maybe discuss specific cases in the next sub header.

I think also clarifying the difference between what is referred to here as "component with message TTL" and "component with native TTL" would help better understand the sections further down.

Reply:
Done.